src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -249,8 +251,20 @@ def _decide_cancel_submission(upload_group: AssetUploadGroup) -> bool:
     is_flag=True,
     help="Allows user to choose Bundle and adds a 'Load a different job bundle' option to the Job-Specific Settings UI",
 )
+@click.option(
+    "--output",
+    type=click.Choice(
+        ["verbose", "json"],
```

Comment:
I'm not sure verbose makes sense here. Any reason to not just do json as choice?

Verbose is usually implied to have a non-verbose mode which is usually the default.

Reply:
Are you proposing that we remove this option altogether and always provide output in JSON format?

Reply:
I originally had it like you suggested and I think it makes sense for this particular use case. I chatted with @ddneilson about it though and he pointed out that over time, we might add `json` output options to the rest of the CLI commands, and someone might expect the output format to not change the type of data being returned, which seemed reasonable.

Reply:
Looks like we're following this pattern elsewhere in deadline-cloud currently. Does this mean you can't select verbose AND json? My thoughts led me to `--output-format JSON` or something. 

My biggest gripe is that for most CLIs I've every used, verbose is usually a standalone flag rather than a choice on individual commands. 

We're not changing this here though, so the change is fine with me.